pkg/project/project.go

```diff
@@ -162,3 +162,147 @@ func Exists(client *occlient.Client, projectName string) (bool, error) {
 	}
 	return false, errors.Errorf(" %v project does not exist", projectName)
 }
+
+type Project struct {
+	Name   string
+	Active bool
+	Client *occlient.Client
```

Comment:
Since, client is not really a property of project it doesn't deserve to be in the project struct in my opinion

Reply:
I agree that the client shouldn't be here. But I can't think of any other way to wrap the client and it is needed so that we don't pass the client in every function call. @girishramnani @anmolbabu @metacosm Please suggest a better design